signal_scanner_bot/verify_util.py

```diff
@@ -0,0 +1,28 @@
+import logging
+import re
+from . import signal
+from typing import List
+from typing import Tuple
```

Comment:
We usually try to group imports like so:
```python
# system library imports

# third party imports

# local imports
```
and within those groups, it's usually `import <x>` before `from <x> import <y>`, and also alphabetical:

```suggestion
import logging
import re
from typing import List
from typing import Tuple

from . import signal
```